### PR TITLE
[8.9] CNVM dash page

### PR DESCRIPTION
**Pull request by benironside**
8.9 introduces a new beta dashboard that shows data from the CNVM integration.

This PR documents the new dashboard on a new page, and makes several minor related changes. 

@nick-alayil please confirm the requirements section is accurate, and let me know if I missed anything!

Preview: [Cloud native vulnerability management dashboard](https://security-docs_3553.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-dashboard.html)

### CODE REVIEW

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[[vuln-management-dashboard]]
+= Cloud Native Vulnerability Vulnerability Management Dashboard
```
Comment (nick-alayil) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:2]: Typo two Vulnerability keywords

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+--
+
+[[CNVM-dashboard-UI]]
+== CNVM dashboard UI
+At the top of the dashboard, 5 summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop down menu.
+
+The page also includes three tables:
+
+* *Top 10 vulnerable resources* shows your VMs with the highest number of vulnerabilities.
+* *Top 10 patchable vulnerabilities* shows the vulnerabilities that were found most often in your environment and can be fixed by a software update.
```
Comment (nick-alayil) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:23]: how about changing most often to most common as we're using the same word in the below item.

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+.Requirements
+[sidebar]
+--
+* The CNVM dashboard is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+--
+
+[[CNVM-dashboard-UI]]
+== CNVM dashboard UI
+At the top of the dashboard, 5 summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop down menu.
```
Comment (nick-alayil) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:18]: It'll be great to highlight the trend graph can be filtered/hide by clicking the severities listed on the right hand side of the graph. Also, the the max duration of the graph is going to be 30 days.

**Review by nick-alayil — APPROVED** [review]
LGTM. Left few comments for review. Thanks @benironside !!

**Comment by natasha-moore-elastic** [comment]
Should we update the screenshot on [this page](https://www.elastic.co/guide/en/security/current/dashboards-overview.html#dashboards-overview), since there's now a new tile for the CNVM dashboard on the Dashboards page in Kibana?

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc (hunk 3 above)
Comment (natasha-moore-elastic) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:18]:
```suggestion
At the top of the dashboard, five summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop down menu.
```

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+[[vuln-management-dashboard]]
+= Cloud Native Vulnerability Vulnerability Management Dashboard
+
+The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of the vulnerabilities detected in your cloud infrastructure by the CNVM integration.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:4]: Correct me if I'm wrong, but isn't installing the CNVM integration a pre-req for using this dashboard? If it is, I recommend moving that information to the requirements block [below](https://github.com/elastic/security-docs/pull/3553/files#diff-77efc2779f8dae31fd438b1a7e66da6edff45c816b825d4586825d66c5c26aabR10) and delete the note on line 6. 
 
```suggestion
The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of your cloud infrastructure vulnerabilities.
```

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/cloud-native-security/vuln-management-dashboard.asciidoc:18]: Just one more tiny edit:

```suggestion
At the top of the dashboard, five summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop-down menu.
```

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+== CNVM dashboard UI
+At the top of the dashboard, 5 summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines, and vulnerabilities (grouped by severity). The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. To view data from specific cloud accounts on this graph, select the account names from the *Accounts* drop down menu.
+
+The page also includes three tables:
+
+* *Top 10 vulnerable resources* shows your VMs with the highest number of vulnerabilities.
+* *Top 10 patchable vulnerabilities* shows the vulnerabilities that were found most often in your environment and can be fixed by a software update.
+* *Top 10 vulnerabilities* shows the most common vulnerabilities in your environment, with additional details.
+
+Each table includes a *View all vulnerabilities* button that opens the <<vuln-management-findings, Vulnerabilities Findings>> page, where you can view additional details.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:26]: Super tiny nit here - I don't think the **View all vulnerabilities** link counts as button in the UI. 

```suggestion
At the bottom of each table, click *View all vulnerabilities* to open the <<vuln-management-findings, Vulnerabilities Findings>> page, where you can view additional details.
```

**Review by nastasha-solomon — APPROVED** [review]
Left some minor suggestions and the dashboard screenshot [here](https://www.elastic.co/guide/en/security/8.8/es-ui-overview.html#_dashboards) will also need to be refreshed. Other than that, everything looks good - thanks!

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+The Cloud Native Vulnerability Management (CNVM) dashboard gives you an overview of vulnerabilities detected in your cloud infrastructure.
+
+image::images/vuln-management-dashboard.png[The CNVM dashboard]
+
+.Requirements
+[sidebar]
+--
+* To collect this data, install the <<vuln-management-get-started, Cloud Native Vulnerability Management>> integration.
+* The CNVM dashboard is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
```
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:19]:
```suggestion
* The CNVM dashboard is available to all Elastic Cloud users. For on-premises deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
```

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+--
+
+beta[]
+
+[[CNVM-dashboard-UI]]
+== CNVM dashboard UI
+At the top of the dashboard, five summary cards appear. They display the number of monitored cloud accounts, scanned virtual machines (VMs), and vulnerabilities (grouped by severity).
```
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:26]: Kind of a personal preference, but I avoid numbering parts of the UI — devs will add or remove things in the future, and it's easy detail to overlook. This also streamlines a little:
```suggestion
The summary cards at the top of the dashboard display the number of monitored cloud accounts, scanned virtual machines (VMs), and vulnerabilities (grouped by severity).
```

File: docs/cloud-native-security/vuln-management-dashboard.asciidoc [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+The *Trend by severity* bar graph complements the summary cards by displaying the number of vulnerabilities found on your infrastructure over time, sorted by severity. It has a maximum time scale of 30 days.
+
+.Graph tips
+[sidebar]
```
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-dashboard.asciidoc:31]: Clever use of the sidebar — I think it works well!

**Review by joepeeples — COMMENTED** [review]
A few small comments, LGTM!

**Pull request merged**